.NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs

```diff
@@ -80,14 +80,27 @@ public NumberRangeModel GetNumberRangeModel(string culture = null, bool fallback
 
         protected override void InitializeConfiguration()
         {
-            /*
             RegisterModel<NumberModel>(
                 Culture.Arabic,
                 (options) => new NumberModel(
                     AgnosticNumberParserFactory.GetParser(AgnosticNumberParserType.Number, new ArabicNumberParserConfiguration(
                                                               new BaseNumberOptionsConfiguration(Culture.Arabic, options))),
                     Arabic.NumberExtractor.GetInstance(NumberMode.PureNumber, options)));
-            */
+
+            RegisterModel<OrdinalModel>(

```

Comment:
I assume this means most tests now pass in .NET for all three types?
If so, please update the PR description with the pass/fail(skipped) numbers per model, ok?
Otherwise, please un-register the models.